test/data/tokenize.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+
+from howl.data.tokenize import Vocab, VocabTrie, WakeWordTokenizer
+
+
+class TestVocab(unittest.TestCase):
```

Comment:
Thanks for the tests! Maybe we can add a `.travis.yml` file to run CI for us like in Anserini? (https://github.com/castorini/anserini/blob/master/.travis.yml)

Reply:
I have a separate ticket for CI  #60

Reply:
Oh I see thanks